src/controllers/queries/index.ts

```diff
@@ -325,6 +325,7 @@ export class QueriesController extends HTMLController {
       emissions?: string
       partialQuery?: 'on'[]
       partialSelect?: 'on'[]
+      [key: string]: unknown
```

Comment:
what why???

Reply:
it has been replaced with `connections: string[]`